azerothcore/azerothcore-wotlk
# fix(Scripts/ZulGurub): Improvements to Gah'zranka:

Summoned Mudskunk Lure object and added visual splash.
Added pathing to Gahzranka
Corrected visual coordinates.
Gahzranka can be summoned only once per instance.
Fixes #11560



## Issues Addressed:

- Closes #11560

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

`.additem 19975 5`
`.additem 19974`
`.go xyz -11695 -1770 13 309` Teleports to Pagle's Pointe
Use Mudskunk Lure and observe Gahz'ranka spawning


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
